### ISSUE TITLE
Test gear transactions

### ISSUE THREAD

**Issue by estensen** [ExcursionClub/ExCSystem#42]
TODO
- [x] Test successful checkout
- [x] Test checkout to unauthorized member
- [ ] Test return date before checkout date
- [x] Test member try to authorize checkout
- [x] Test checkout gear that doesn't exist
- [x] Test checkout gear that are already checked out
- [x] Test return gear
- [x] Test return gear that are already returned

**Comment by sarahwells12595** [comment]
Test unauthorized user trying to checkout gear.
https://github.com/ExcursionClub/ExCSystem/commit/5e3f7120ed80bf1a307bc2ecbdbc368c420f1bf8

**Comment by sarahwells12595** [comment]
https://github.com/ExcursionClub/ExCSystem/commit/360b3585b765ecd09f662b2425539d3d70480065

**Comment by sarahwells12595** [comment]
https://github.com/ExcursionClub/ExCSystem/pull/57

**Comment by estensen** [comment]
@sarahwells12595 remember to only close an issue once the code is merged into the master branch
